src/scripts/party/PartyPokemon.ts

```diff
@@ -53,16 +53,16 @@ class PartyPokemon implements Saveable {
         category = 0,
         pokerus = false
     ) {
-        this.proteinsUsed = ko.observable(proteinsUsed);
-        this.effortPoints = ko.observable(effortPoints);
-        this._breeding = ko.observable(breeding);
-        this._shiny = ko.observable(shiny);
-        this._level = ko.observable(1);
-        this._attackBonusPercent = ko.observable(attackBonusPercent);
-        this._attackBonusAmount = ko.observable(attackBonusAmount);
-        this._attack = ko.observable(this.calculateAttack());
-        this._category = ko.observable(category);
-        this._pokerus = ko.observable(pokerus);
+        this.proteinsUsed = ko.observable(proteinsUsed).extend({ numeric: 0 });
+        this.effortPoints = ko.observable(effortPoints).extend({ numeric: 0 });
+        this._breeding = ko.observable(breeding).extend({ boolean: null });
+        this._shiny = ko.observable(shiny).extend({ boolean: null });
+        this._level = ko.observable(1).extend({ numeric: 0 });
+        this._attackBonusPercent = ko.observable(attackBonusPercent).extend({ numeric: 0 });
+        this._attackBonusAmount = ko.observable(attackBonusAmount).extend({ numeric: 0 });
+        this._attack = ko.observable(this.calculateAttack()).extend({ numeric: 0 });
+        this._category = ko.observable(category).extend({ numeric: 0 });
+        this._pokerus = ko.observable(pokerus).extend({ boolean: null });
```

Comment:
This one is not going to be boolean with the pokerus enum PR, do you want to leave it out of this one?

Reply:
I think we still want to keep it, so we remember to change it when we get a merged conflict

Reply:
I don't think it will create a conflict, because nothing has changed on this line for the other PR.
I'll remove it and create a commit on the other PR to update it.
```suggestion
        this._pokerus = ko.observable(pokerus);
```